pkg/features/kube_features.go

```diff
@@ -391,10 +391,14 @@ const (
 	// Enable POD resources API with Get method
 	KubeletPodResourcesGet featuregate.Feature = "KubeletPodResourcesGet"
 
-	// KubeletSeparateDiskGC enables Kubelet to garbage collection images/containers on different filesystems
 	// owner: @kannon92
 	// kep: https://kep.k8s.io/4191
 	// alpha: v1.29
+	// alpha: v1.31
```

Comment:
```suggestion
	// beta: v1.31
```

Reply:
Ahh. Doh. Well spotted. It has been a long day :smile:

Reply:
Done.